examples/mnist/quantized.cpp

```diff
@@ -29,7 +29,7 @@ void construct_net(network<sequential> &nn) {
   // C : convolution
   // S : sub-sampling
   // F : fully connected
-  nn << quantized_convolutional_layer<tan_h>(
+  nn << quantized_convolutional_layer(
           32, 32, 5, 1, 6, padding::valid, true, 1, 1,
           backend_type)  // C1, 1@32x32-in, 6@28x28-out
```

Comment:
Ago save line spaces maybe `convq` or similar instead of `quantized_convolutional_layer`